pgtest.go

```diff
@@ -39,21 +39,24 @@ type PG struct {
 // crashes, but we don't care about that anyway during unit testing.
 //
 // Use the DB field to access the database connection
-func Start() (*PG, error) {
-	return start(false, "")
+func Start(config ...*PGConfig) (*PG, error) {
```

Comment:
I'd leave this method unchanged and put a `Start()` method on the config object (see my original comment).

This version is confusing: what happens if you pass multiple configs? I know it's used here as a hack to have optional arguments, but it's not very clean.

Reply:
Putting the start method on the config object also makes things much less verbose.

Instead of:
```go
pgtest.Start(pgtest.NewConfig().Persistent())
```
It could be:
```go
pgtest.New().Persistent().Start()
```

Cleaner, more readable and less stuttering (no `pgtest` twice).